qiita_pet/handlers/study_handlers/tests/test_artifact.py

```diff
@@ -140,6 +140,33 @@ def test_delete_artifact(self):
         wait_for_prep_information_job(1)
 
 
+class ArtifactGetSamplesTest(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/artifact/samples/', {'ids[]': [4, 5]})
+        self.assertEqual(response.code, 200)
+        exp = (
+            '{"status": "success", "msg": "", "data": {"4": ['
```

Comment:
thanks for the explanation, however, this works fine in the tests in travis and my laptop ... thus, not sure if needs to be changed, should I?